src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,340 @@
+          );
+          window.addEventListener(
+            'scroll',
+            debounce(this.$_handleAbortTimer, scrollDebounceMs),
+          );
+          this.$_raiseAnalyticsEvent('Wake Lock Toggle Enabled');
+        } catch (e) {
+          this.isChecked = false;
+        }
+      },
+      $_releaseWakeLock: function() {
+        try {
+          this.wakeLock.release();
+          this.wakeLock = null;
+          //Clear timer
+          this.$_handleAbortTimer();
+          //Clean up event listeners
+          document.removeEventListener(
+            'visibilitychange',
+            this.$_handleVisibilityChange,
+          );
+          window.removeEventListener(
+            'scroll',
+            debounce(this.$_handleAbortTimer, scrollDebounceMs),
+          );
+        } catch (e) {
+          console.log(`Wake Lock Release Error: ${e.name}, ${e.message}`);
+        }
+      },
+      $_handleAbortTimer: function() {
+        //If there is an existing timer then clear it and set to zero
+        if (this.timerId !== 0) {
+          clearTimeout(this.timerId);
+          this.timerId = 0;
+        }
+        //Start new timer; Will be triggered from toggle enabled or scroll event
+        if (this.isChecked) {
+          this.timerId = setTimeout(
+            this.$_releaseWakeLock,
+            this.settingsInternal.timeoutDurationMs,
+          );
+        }
+      },
+      $_handleVisibilityChange: function() {
+        //Handle navigating away from page/tab
+        if (this.isChecked) {
+          this.$_releaseWakeLock();
+          this.isChecked = false;
+        }
+      },
+      $_raiseAnalyticsEvent: function(eventType) {
+        let eventParams = {
+          EventType: eventType,
+          Position: window.location.pathname || '',
+        };
+        Analytics.raiseEvent(eventParams);
+      },
+    },
+  };
+</script>
+```
+
+### Results
+
+The Vue.js component has been deployed on all three sites and delivered great results.
+During the period from December 10th, 2019 to January 10th, 2020,
+BettyCrocker.com reported the following metrics:
+
+- Of all Betty Crocker users with a browser compatible with the Wake Lock API,
+  3.5% of them enabled the feature immediately, making it a top-5 action.
+- The session duration for users who enabled the wake lock was 3.1× longer
+  than for users who did not.
+- The bounce rate for users who enabled the wake lock was 50% lower
+  for those using the wake lock feature.
+- Indicators of purchase intent were about 300% higher for wake lock users compared to all users.
+
+## Conclusions
+
+Betty Crocker has seen fantastic results using the Wake Lock API.
+You can test the feature yourself by searching for your favorite recipe on any of their sites
+([BettyCrocker](https://www.bettycrocker.com/),
+[Pillsbury](https://www.pillsbury.com/), or [Tablespoon](https://www.tablespoon.com/))
+and toggling the "Prevent your screen from going dark while you cook" toggle.
```

Comment:
```suggestion
and enabling the **Prevent your screen from going dark while you cook** toggle.
```

Switching to `enabling` avoids having to use the word `toggle` redundantly

Style guide convention is to format UI labels in bold: https://developers.google.com/style/ui-elements#formatting